File: ReportServiceUsingCsharp/Dto/ReportResultDto.cs
using System.Text.Json.Serialization;

namespace ReportServiceUsingCsharp.Dto;

public class ReportResultDto
{
    [JsonPropertyName("user_id")]
    public string UserId { get; }

    [JsonPropertyName("count_sign_in")]
    public string CountSignIn { get; }

    public ReportResultDto(string userId, string countSignIn)
    {
        UserId = userId;
        CountSignIn = countSignIn;
    }
}

File: reportservice/src/main/java/dev/sverdlov/reportservice/dto/ReportResultDto.java
package dev.sverdlov.reportservice.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ReportResultDto {
    @JsonProperty("user_id")
    private final String userId;

    @JsonProperty("count_sign_in")
    private final String countSignIn;

    public ReportResultDto(String userId, String countSignIn) {
        this.userId = userId;
        this.countSignIn = countSignIn;
    }

    public String getUserId() {
        return userId;
    }

    public String getCountSignIn() {
        return countSignIn;
    }
}